### PR TITLE
[Feature] Enhance metadata table info module style

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

Enhance metadata TableInfo module table and pagination

image

### CODE REVIEW

**Review by labbomb — APPROVED** [review]
LGTM

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+      {
+        title: 'Nullable',
+        key: 'nullAble',
+        align: 'center',
+        render(rowData) {
+            return <n-checkbox v-model:checked={rowData.nullAble} />
+        },
+      },
+      {
+        title: 'Pk',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:84]:
```suggestion
        title: 'Primary Key',
```

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        key: 'pk',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:85]:
```suggestion
        key: 'primaryKey',
```

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        align: 'center',
+        render(rowData) {
+          return <n-checkbox v-model:checked={rowData.pk} />
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:88]: Ditto.

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        key: 1,
+        columnName: 'Name',
+        dataType: 'STRING',
+        nullAble: false,
+        pk: true,
+        partitionKey: false
+      },
+      {
+        key: 2,
+        columnName: 'Desc',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:57]:
```suggestion
        columnName: 'Description',
```

**Review by SteNicholas — CHANGES_REQUESTED** [review]
@zhangmo8, thanks for contribution. I left a few comments for this pull request. PTAL.

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+      {
+        title: 'Description',
+        key: 'description',
+        align: 'center',
+        render(rowData) {
+            return rowData.defaultValue || '-'
+        },
+      },
+      {
+        title: 'Operate',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:116]:
```suggestion
        title: 'Operation',
```

File: paimon-web-ui-new/src/views/metadata/components/table/index.tsx [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+        key: 'actions',
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/table/index.tsx:117]:
```suggestion
        key: 'operation',
```

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @zhangmo8 updates.

**Pull request merged**